src/mixins/localize-base-mixin.ts

```diff
@@ -88,7 +88,11 @@ export const localizeBaseMixin = (superClass) =>
           argObject[args[i]] = args[i + 1];
         }
 
-        return translatedMessage.format(argObject);
+        try {
+          return translatedMessage.format(argObject);
+        } catch (err) {
+          return "Translation " + err;
```

Comment:
What kind of errors would this generate ? And how would that look in the UI?

Reply:
![image](https://user-images.githubusercontent.com/5662298/47734091-f8e16d00-dc69-11e8-85b4-277818532dff.png)

Reply:
https://github.com/yahoo/intl-messageformat/blob/0cecaff663d2d73bf6602b71fab3b5e01d0ca16f/src/core.js#L49

Reply:
![image](https://user-images.githubusercontent.com/5662298/47734669-0d723500-dc6b-11e8-8981-a6193feae996.png)

Reply:
It ain't pretty, but better than not showing anything at all, and it is an error after all... (we might make it red)

Reply:
We can't make it red as we only can return strings.

I guess it's fine.

Reply:
Also someone stop that Dutch translator …